### PR TITLE
The History Behind When Build Becomes Buy

### PR DESCRIPTION
**Pull request by brunokrebs**

### CODE REVIEW

File: _posts/2017-02-24-the-history-behind-when-build-becomes-buy.markdown [hunk 1]
```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: "The History Behind When Build Becomes Buy"
+description: "Although considered the easiest option, building has it limitations when compared to the benefits of buying."
+date: 2017-02-24 10:02
+category: Growth, Generic
```
Comment (brunokrebs) [inline _posts/2017-02-24-the-history-behind-when-build-becomes-buy.markdown]: @diegopoza could you please check the correct category? I couldn't identify.

File: _posts/2017-02-24-the-history-behind-when-build-becomes-buy.markdown [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+author:
+  name: Diego Poza
+  url: https://twitter.com/diegopoza
+  avatar: https://avatars3.githubusercontent.com/u/604869?v=3&s=200
+  mail: diego.poza@auth0.com
+design:
+  bg_color:
```
Comment (brunokrebs) [inline _posts/2017-02-24-the-history-behind-when-build-becomes-buy.markdown]: Gonna ask Barbu for those

File: _posts/2017-02-24-the-history-behind-when-build-becomes-buy.markdown [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+Anyone can now trust their compliance, authentication and security needs with a specialized provider. They don't have to worry about exposing themselves to vulnerabilities because of the risks involved with building their own system.
+
+Nor do they have to worry about the cost of revisions and upgrades for an in-house product. These can be expensive, time-consuming to handle and require constant involvement, especially as a business grows.
+
+## To Buy or Not to Buy
+
+Products and services for enterprises and startups have come a long way in many areas of technology. While it used to make sense to build in-house, customer needs and expectations have become more complex. This change means technology has to adjust accordingly.
+
+You don't have to use your precious time trying to figure out something if you buy a third-party solution. Even if you have the capability and resources, chances are there's something you haven't thought of.
```
Comment (brunokrebs) [inline _posts/2017-02-24-the-history-behind-when-build-becomes-buy.markdown]: The author started this paragraph with:

>You don't have to use your precious time trying to figure out something if you **build on your own**.

And I changed to:

>You don't have to use your precious time trying to figure out something if you **buy a third-party solution**.

As it made more sense to me. Could someone check which one is correct? @diegopoza @kukicado @kmaida

Comment (diegopoza) [inline _posts/2017-02-24-the-history-behind-when-build-becomes-buy.markdown]: Yes, what you changed is correct @brunokrebs

**Pull request merged**